examples/3d/deterministic.rs

```diff
@@ -26,16 +26,22 @@ fn setup(
     deterministic_rendering_config.stable_sort_z_fighting = true;
 
     // Help message will be rendered there.
-    commands.spawn(TextBundle::default());
+    commands.spawn(TextBundle::default().with_style(Style {
+        position_type: PositionType::Absolute,
+        top: Val::Px(12.0),
```

Comment:
Mildly amused that half of examples that do this use 12px and half use 10px.

Reply:
I'll fix that someday... the officially Cart approved margin is 12px: https://github.com/bevyengine/bevy/pull/8478#issuecomment-1523964321